lib/configuration_methods.rb

```diff
@@ -50,11 +50,11 @@ def show_service?(service, user)
         self["services.#{service}.authorized"] == true
     end
 
-    def local_posts_stream?(user)
+    def local_posts_stream?(person)
       return true if settings.enable_local_posts_stream == "admins" &&
-                     Role.is_admin?(user)
+                     Role.is_admin?(person)
       return true if settings.enable_local_posts_stream == "moderators" &&
-                     (Role.moderator?(user) || Role.is_admin?(user))
+                     (Role.moderator?(person) || Role.is_admin?(person))
```

Comment:
`Role.is_admin?` isn't needed here, since `Role.moderator?` includes admins.

But also, since this `local_posts_stream?` is only relevant for the user and not for persons, you can keep `user` as parameter, and then use `user.moderator?` here (which also includes admins) and `user.admin?` above.

Reply:
Ok. By the way.. is there a reason the "role" is linked to the person rather than the to local user?

Reply:
Roles are also used for community spotlights, which can be people from other pods which don't have a user object.

Reply:
Hm, isn't this also not available in the UI? 
But OK for now, thanks.